Cargo.lock

```diff
@@ -1382,6 +1391,16 @@ version = "0.1.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "b15813163c1d831bf4a13c3610c05c0d03b39feb07f7e09fa234dac9b15aaf39"
 
+[[package]]
+name = "packed_simd_2"
```

Comment:
Don't be confused by the [README on crates.io](https://crates.io/crates/packed_simd_2), as I nearly was. `packed_simd` is the unmaintained version; `packed_simd_2` is the (somewhat) maintained one. https://github.com/dalek-cryptography/curve25519-dalek/pull/520, which has been merged but not released, will remove this dependency. So, we won't be relying on it long-term.